src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,382 @@
+
+```
+helm upgrade --install newrelic newrelic/nri-bundle \
+--set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+--set global.cluster=K8S_CLUSTER_NAME \
+--set infrastructure.enabled=true \
+--set prometheus.enabled=true \
+--set webhook.enabled=true \
+--set ksm.enabled=false \
+--set kubeEvents.enabled=true \
+<var>--set global.fargate=true</var> \
+<var>--set newrelic-infra-operator.enabled=true</var>
+```
+
+<Callout variant="important">
+In the case of a hybrid cluster (EC2 and Fargate), make sure after deploying that the solution is not selected by any Fargate profiles; otherwise, the `DaemonSet` instances will be stuck in a pending state. Fargate-only environments don't have this limitation because no `DaemonSet` instances are created.
+
+</Callout>
+
+#### Configuration [#config-auto]
+
+You can configure different aspects of the automatic injection. By default, the operator will inject the monitoring sidecar to all pods deployed in Fargate nodes which are not part of a `Job` or a `BatchJob`.
+
+Using configurations, you can change this behavior. For example, you can define selectors to narrow or widen the selection of pods that are injected, assign resources to the operator, and tune the sidecar. Also, you can add other attributes, labels, and environment variables. Please refer to the chart [README.md](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infra-operator/README.md) and [values.yaml](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infra-operator/values.yaml).
+
+<Callout variant="tip">
+Injecting the sidecar into deployments scheduled into EC2 nodes may lead to incorrect or duplicate data, especially if those nodes are already being monitored with the `DaemonSet`. Take this into account when designing custom selectors for the automatic injection.
+</Callout>
+
+#### Update to the latest version or to a new configuration [#update-auto-install-version]
+
+To update to the latest version of the EKS Fargate integration, upgrade the chart and reinstall the bundle by simply running again the command above.
+
+To update the configuration of the infrastructure agent injected or the operator itself, modify the `values.yaml` and upgrade the Helm release with the new configuration. The operator is updated immediately. However, another rollout of the workloads is needed in order to inject the infrastructure agent sidecar with the new configuration.
+
+```
+  $ kubectl rollout restart deployment <var>YOUR_DEPLOY</var>
+```
+
+#### Uninstall the Fargate integration [#uninstall-auto-injection]
+
+We recommend that you uninstall the infrastructure operator that performs the automatic injection but keep the rest of the New Relic solution. Using Helm, disable the infra-operator by setting `infra-operator.enabled` to `false`, either in the `values.yaml` file or in the command line (`--set`), and re-run the upgrade (`--install`) command above.
```

Comment:
```suggestion
In order to uninstall the sidecar performing the automatic injection but keep the rest of the New Relic solution, using Helm, disable the infra-operator by setting `infra-operator.enabled` to `false`, either in the `values.yaml` file or in the command line (`--set`), and re-run the installation command above.

We strongly recommend to keep the `--set global.fargate=true` flag, since it does not enable automatic injection but makes other components of the installation Fargate-aware, preventing unwanted behavior.
```